src/index.js

```diff
@@ -555,9 +556,20 @@ const Instauto = async (db, browser, options) => {
       if (!foundClickable) throw new Error('Like button not found');
 
       if (!dryRunIn) {
-        foundClickable.click();
+        const img = dialog.querySelectorAll('._aagt')[0];
```

Comment:
I think these classes `_aagt` and `_a9zs` are not very safe to use. will probably change very often